build/docker-compose.net8.0.yml

```diff
@@ -6,4 +6,4 @@ services:
       args:
         PUBLISH_FRAMEWORK: net8.0
         TEST_SDK_VERSION: "8.0"
-        BUILD_SDK_VERSION: "8.0"
+        BUILD_SDK_VERSION: "9.0"
```

Comment:
This would fail for now, both for `confluentkafka-integration` and `stackexchangeredis-integration`. The current latest SDK in `mcr.microsoft.com/dotnet/sdk:9.0` is `9.0.100-rc.2.24474.11`, as shown in CI.

I'll revert global.json to use rc.2 for now.

Reply:
Updated `global.json` now that the docker image is updated: https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/2316/commits/1c0564573b91a4e529951a6985d10372bc37f7cf

```
docker run --rm mcr.microsoft.com/dotnet/sdk:9.0 dotnet --info
.NET SDK:
 Version:           9.0.100
 Commit:            59db016f11
 Workload version:  9.0.100-manifests.3068a692
 MSBuild version:   17.12.7+5b8665660
```